modules/core/src/parallel.cpp

```diff
@@ -455,6 +455,29 @@ void cv::parallel_for_(const cv::Range& range, const cv::ParallelLoopBody& body,
     }
 }
 
+#if __cplusplus >= 201103L
+class ParallelLoopBodyLambdaWrapper : public cv::ParallelLoopBody
```

Comment:
But in this case user should use strange wrapper instead of directly pass lambda to `parallel_for_`. Most users can write such a dummy wrapper by themselves, so we don't need this overload at all.
@vpisarev what do you think?